packages/components/package.json

```diff
@@ -21,7 +21,7 @@
     "start": "yarn tsc --watch",
     "start:storybook": "start-storybook",
     "typecheck": "tsc --noEmit --pretty",
-    "chromatic": "if-env CIRCLE_BRANCH=master && yarn chromatic:master || yarn chromatic:branch",
+    "chromatic": "!(git diff --quiet ./src) && (if-env CIRCLE_BRANCH=master && yarn chromatic:master || yarn chromatic:branch)",
```

Comment:
we would probably always want to run master to keep the latest baseline. Should move this to chromatic:branch